internal/acctest/sweep/organizations.go

```diff
@@ -22,10 +22,11 @@ func Organizations() {
 				return err
 			}
 
-			var page int
 			var result *multierror.Error
+			var from string
+
 			for {
-				organizationList, err := api.Organization.List(ctx, management.Page(page))
+				organizationList, err := api.Organization.List(ctx, management.From(from), management.Take(100))
```

Comment:
From the docs:


> Note: The first time you call this endpoint using Checkpoint Pagination, you should omit the from parameter. If there are more results, a next value will be included in the response. You can use this for subsequent API calls. When next is no longer included in the response, this indicates there are no more pages remaining.


https://auth0.com/docs/api/management/v2/organizations/get-organizations